projects/kubernetes/cloud-provider-aws/1-23/GIT_TAG

```diff
@@ -1 +1 @@
-v1.23.6
```

Comment:
Did we really update that many patch versions?

Reply:
Oh, or does it jsut track the k8s version?

Reply:
Internal source of truth is used to determine which patch version to upgrade to for this component.